gtfs/spec/en/reference.md

```diff
@@ -424,14 +441,15 @@ To describe the different fare media that can be employed to use fare products.
 
 File: **Optional**
 
-Primary key (`fare_product_id`, `fare_media_id`)
+Primary key (`fare_product_id`, `rider_category_id`, `fare_media_id`)
 
 Used to describe the range of fares available for purchase by riders or taken into account when computing the total fare for journeys with multiple legs, such as transfer costs.
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-| `fare_product_id` | ID | **Required** | Identifies a fare product or set of fare products.<br><br>Multiple records in [fare_products.txt](#fare_productstxt) may share the same `fare_product_id`, in which case all records with that ID will be retrieved when referenced from another file.<br><br>Multiple records may share the same `fare_product_id` but with different `fare_media_id`s, indicating various methods available for employing the fare product, potentially at different prices. |
+| `fare_product_id` | ID | **Required** | Identifies a fare product or set of fare products.<br><br>Multiple records in [fare_products.txt](#fare_productstxt) may share the same `fare_product_id`, in which case all records with that ID will be retrieved when referenced from another file.<br><br>Multiple records may share the same `fare_product_id` but with different `fare_media_id`s, indicating various methods available for employing the fare product, potentially at different prices.<br><br>Multiple records may share the same `fare_product_id` but with different `rider_categories_id`s, indicating multiple rider categories are eligible for the fare product, potentially at different prices. |
```

Comment:
I think there’s an opportunity here to clarify both existing language and new language around what is permitted with shared `fare_product_id`s. Would the following language be an improvement, and does it accurately describe what is allowed?

> Multiple records sharing the same `fare_product_id`  are permitted as long as they contain different `fare_media_id`s or `rider_category_id`s. Differing `fare_media_id`s would indicate various methods are available for employing the fare product, potentially at different prices. Differing `rider_category_id`s would indicate multiple rider categories are eligible for the fare product, potentially at different prices.

Reply:
Thanks for the suggestion @westontrillium!
This looks great to me - changes on 8068e8c